config/locales/ja.yml

```diff
@@ -142,17 +113,15 @@ ja:
     Rich_Harris:
       title_html: '%{project1_url}と%{project2_url}のメンテナ'
       name_html: ' Rich Harris (%{url})'
-      message_html: "The thing that most open source projects lack is not money, but time. Many maintainers have full-time jobs and can't realistically cut back their hours. What almost all projects need is more contributors. Open Source Friday is a perfect way for companies to support the open source ecosystem they depend on in a meaningful way."
+      message_html: "多くのOSSプロジェクトでは、資金ではなく掛ける時間が不足しています。メンテナの多くは、フルタイムの仕事をしていて、その時間を削ることが難しい状況です。全てのプロジェクトで必要とされるのは、貢献者です。オープンソース フライデーは、事業が依存しているOSSエコシステムへ意味あるサポートを行う、企業にとって完璧な方法です。"
     Brett_Cannon:
       title_html: '%{project_url}開発者'
       name_html: 'Brett Cannon (%{url})'
       message_html: 'Once you are a maintainer on a project that others rely on, open source goes from "sharing the code" to "sharing the maintenance". As we all know, "maintenance" takes time which is most effectively spent in a continuous block to help make sure you can finish that e.g. code review before you start to lose context.'
-
-  # app/views/users/show.rb
   users:
     welcome: "オープンソース フライデーに参加しました!"
-    joined: "はオープンソース フライデーに参加しました!"
-    not_joined:  "は、オープンソース フライデーに（まだ）参加してません。"
+    'joined:': "オープンソース フライデーに参加しました!"
```

Comment:
Here I find a changing key from 'joined' to 'joined:' Are there any bugs or problems in processing YAML and translation file for transifex?

Reply:
@miurahr Looks like the issue was in the original `en.yml` file: https://github.com/MikeMcQuaid/ossfriday/blob/074ea597ba39b4d3b74fee06c343dfb5d0f65985/config/locales/en.yml#L155

Going to reupload `en.yml` and redownload `ja.yml`.

Reply:
Fixed in https://github.com/ossfriday/ossfriday/pull/198